.ci/test-run-scheduler-schema.sql

```diff
@@ -0,0 +1,148 @@
+create type test_run_status as enum ('running', 'passed', 'failed');
+
+create table busted_test_run_request
+(
+    id           serial primary key,
+    workflow_id  bigint    not null,
+    created_at   timestamp not null default current_timestamp,
```

Comment:
```suggestion
    created_at   timestamp with time zone not null default current_timestamp at time zone 'UTC',
```

If `second` precision is enough you can use `current_timestamp(0)` (or current_timestamp(3) if milliseonds, but full precision is ok too)